app.rb

```diff
@@ -0,0 +1,189 @@
+$LOAD_PATH << '.'
+require 'classroom'
+require 'person'
+require 'teacher'
+require 'student'
+require 'book'
+require 'rental'
+require 'utils'
```

Comment:
- [ ] Good job so far 👍 You do not have any files named `utils` in your directory, Kindly correct this name to `main` as I believe that is the file you are trying to access.

Reply:
resolved 👍